src/app.ts

```diff
@@ -35,23 +47,161 @@ const app = new Koa();
 app.use(cors());
 app.use(bodyParser());
 
-const publicRouter = new Router().get("/api/operations", async ctx => {
-  // TODO: Do we need to paginate?
-  const ops = await operations.orderBy("created_at").get();
-  const parsedOps: Array<Object> = [];
-  ops.forEach(op =>
-    parsedOps.push({
-      id: op.id,
-      creator_mid: op.get("creator_mid"),
-      creator_uid: op.get("creator_uid"),
-      op_code: op.get("op_code"),
-      data: op.get("data")
-    })
+async function validateUid(uid, ctx, next) {
+  const uidDoc = await members.doc(uid).get();
+  if (!uidDoc.exists) {
+    ctx.status = 404;
+    return;
+  }
+  ctx.state.toMember = uidDoc;
+  return await next();
+}
+
+function getPrivateVideoRef(memberUid): Storage.File {
+  return storage
+    .bucket(PRIVATE_VIDEO_BUCKET)
+    .file(`private-video/${memberUid}/invite.mp4`);
```

Comment:
Is this what makes us not support `.mov` files and stuff? any way we can make this more flexible?

Reply:
yerp, wanted to save for a separate PR. That would require a change on the frontend side too anyways atm.